packages/redux/src/recentlyViewed/actions/factories/removeRecentlyViewedProductFactory.ts

```diff
@@ -0,0 +1,45 @@
+import * as actionTypes from '../../actionTypes';
+import type { DeleteRecentlyViewedProducts } from '@farfetch/blackout-client/src/recentlyViewed/types';
+import type { Dispatch } from 'redux';
+import type { RemoveRecentlyViewedProductAction } from '../../types';
+import type { RemoveRecentlyViewedProductsFactory } from './types';
+
+/**
+ * Method to create a thunk factory configured with the specified client for deleting a recently viewed product, both locally and on the server.
+ *
+ * @param deleteRecentlyViewedProduct - Delete recently viewed client.
+ *
+ * @returns Thunk factory.
+ */
+
+const removeRecentlyViewedProductFactory: RemoveRecentlyViewedProductsFactory<DeleteRecentlyViewedProducts> =
+
```

Comment:
Same as https://github.com/Farfetch/blackout/pull/113/files#r783181459

Reply:
Same case as previous one, when format on saving, adds this line automatically